zsh-nvm.plugin.zsh

```diff
@@ -102,6 +102,7 @@ _zsh_nvm_lazy_load() {
     eval "$cmd(){
       unset -f $cmds > /dev/null 2>&1
       _zsh_nvm_load
+      [[ \"$NVM_AUTO_USE\" == true ]] && add-zsh-hook chpwd _zsh_nvm_auto_use
```

Comment:
Why do we need to setup the `chpwd` hook here?

Shouldn't it already be set here?

https://github.com/lukechilds/zsh-nvm/pull/55/files#diff-dfc5680d74f94037e00d26cbeba9485eR212

Reply:
The tests all still pass if this line is removed:

https://travis-ci.org/github/lukechilds/zsh-nvm/jobs/687714916

https://github.com/lukechilds/zsh-nvm/commit/30f85550443436f688ecd2baaea0b25653464cdc